scripts.js

```diff
@@ -61,16 +110,32 @@ function savePalette() {
   loadPageColors();
   console.log(savedPalettes)
   renderSavedPalette();
-  // dispplaySavedPalette function invoked here
-  // have saved pallets .locked all changed to one value
+  unlockAllLocks()
 };
 
 function show(element) {
-  element.classList.remove('hidden')
+  element.classList.remove('hidden');
 };
 
 function hide(element) {
-  element.classList.add('hidden')
+  element.classList.add('hidden');
+};
+
+function toggleHiddenClass(element) {
+  element.classList.toggle('hidden');
+}
+
+function unlockAllLocks() {
```

Comment:
Maybe we can utilize the node list of unlocked and locked to refactor this... just an idea for later. Awesome that it's working now!